lib/routes.js

```diff
@@ -100,5 +100,12 @@ export default function routes(req, res, logger) {
             undefined, res, log);
     }
 
+    if ((req.query.partNumber || req.query.uploadId)
```

Comment:
Part numbers can be any number from 1 to 10,000, inclusive.
http://docs.aws.amazon.com/AmazonS3/latest/API/mpUploadUploadPart.html